.github/workflows/ci.yml

```diff
@@ -22,83 +22,54 @@ env:
   KNAPSACK_PRO_LOG_LEVEL: info
 
 jobs:
-  bundle_install:
+  build:
     runs-on: ubuntu-latest
+    env:
+      E2E: true
+      YARN_ENABLE_HARDENED_MODE: 0
```

Comment:
Yarn's hardened mode is still enabled by default in other build, ie `build_test` as part of precompile steps. This is an important security feature but we're shoving it to into later builds for speeds optimization.